actors/miner/tests/precommit_deposit_and_initial_pledge_positive_test.rs

```diff
@@ -38,10 +38,217 @@ fn initial_pledge_clamped_at_one_attofil() {
         &reward_estimate(),
         &power_estimate(),
         &circulating_supply(),
+        // NOTE: setting this to zero preserves the original pledge definition (before baseline bug fix)
```

Comment:
You should probably change the name of this file since the scope of testing has changed signficantly.

Reply:
Or alternatively move these tests to a new file

Reply:
Done.